### PR TITLE
Replace ‘Ubuntu’ with ‘Multipass’ in docs

### PR DESCRIPTION
**Pull request by barrymcgee**

### CODE REVIEW

File: docs/macos-multipass.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 This document describes how to set up a development environment for the Juju
-GUI on macOS with multipass. The *Initial setup* section must be done only once
-initially. The *Working with the GUI* section describes the daily routine of
+GUI on macOS with multipass. The _Initial setup_ section must be done only once
+initially. The _Working with the GUI_ section describes the daily routine of
 switching to GUI branches under review and QAing them.
 
 ## Initial setup
 
-### Create an ubuntu instance
```
Comment (frankban) [inline docs/macos-multipass.md]: In this case this is still an ubuntu instance.

File: docs/macos-multipass.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+### Create a Multipass instance
 
-- Download and install multipass from
+- Download and install Multipass from
   <https://private-fileshare.canonical.com/~msawicz/multipass/current/>.
-- Create an ubuntu instance, for example with
+- Create a Multipass instance, for example with
```
Comment (frankban) [inline docs/macos-multipass.md]: Ditto.

File: docs/macos-multipass.md [hunk 3]
```diff
@@ -84,38 +93,43 @@ rm ~/code/ubuntu/it-works # Clean it up.
...
+
 - Install guiproxy with `go get github.com/juju/guiproxy`.
 - Double check guiproxy is correctly installed with `guiproxy -version`: you
   should have at least version 0.7.5.
 
 ## Working with the GUI
 
 As mentioned, while development can be done using your favorite macOS editor
-(if you wish), the GUI can be run exclusively from ubuntu. So, from now on,
+(if you wish), the GUI can be run exclusively from Multipass. So, from now on,
```
Comment (frankban) [inline docs/macos-multipass.md]: In this context ubuntu is ok.

File: docs/macos-multipass.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+GUI on macOS with multipass. The _Initial setup_ section must be done only once
+initially. The _Working with the GUI_ section describes the daily routine of
 switching to GUI branches under review and QAing them.
 
 ## Initial setup
 
 ### Create an ubuntu instance
 
-- Download and install multipass from
+- Download and install Multipass from
```
Comment (hatched) [inline docs/macos-multipass.md]: This is no longer the correct location for this

**Pull request merged**